### PR TITLE
fix(ivy): pipes should overwrite pipes with later entry in the pipes array

### PR DESCRIPTION
**Pull request by marclaval**
This PR fixes FW-894.

### CODE REVIEW

File: packages/core/src/render3/pipe.ts [hunk 1]
```diff
@@ -55,7 +55,7 @@ export function pipe(index: number, pipeName: string): any {
  */
 function getPipeDef(name: string, registry: PipeDefList | null): PipeDef<any> {
   if (registry) {
-    for (let i = 0; i < registry.length; i++) {
+    for (let i = registry.length - 1; i >= 0; i--) {
```
Comment (petebacondarwin) [inline packages/core/src/render3/pipe.ts]: Is this performance critical? Maybe a compiler might optimize this for you but you could avoid computing the `length-1` each loop by assigning it to a constant.

Comment (gkalpak) [reply packages/core/src/render3/pipe.ts]: B...but...he's not computing it each loop 😕
Or did you mean not to compute it on each `getPipeDef()` call?

**Review by mhevery — APPROVED** [review]

File: packages/core/src/render3/pipe.ts (hunk 1 above)
Comment (petebacondarwin) [reply packages/core/src/render3/pipe.ts]: LOL! Sorry. Not actually reading the code.

**Comment by AndrewKushnir** [comment]
@marclaval could you please rebase from the latest master and resolve conflict (and add "merge" label back)? Thank you.

**Comment by marclaval** [comment]
PR rebased and ready for merge

**Comment by kara** [comment]
[presubmit](http://test/OCL:227896723:BASE:227896755:1546634057794:ea1d07f7)

**Pull request closed without merging**